_overviews/jdk-compatibility/overview.md

```diff
@@ -83,7 +83,7 @@ For possible Scala issues, see the [jdk11](https://github.com/scala/bug/labels/j
 
 Early access builds of JDK 18, a non-LTS release, are already available.
 
-Initial support for JDK 18 is included in Scala 2.13.7 and 2.12.15.
+Initial support for JDK 18 was included in Scala 2.13.7 and 2.12.15.
```

Comment:
Of course, it is still included. We're so proud of English until we encounter certain modes.

Reply:
I also considered "has been included since", but 🤷

Reply:
for 2022, I predict emoticons will experience retro hipster cachet.